packages/aws_signature_v4/example/bin/example.dart

```diff
@@ -0,0 +1,181 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'dart:convert';
+import 'dart:io';
+import 'dart:math';
+
+import 'package:args/args.dart';
+import 'package:aws_common/aws_common.dart';
+import 'package:aws_signature_v4/aws_signature_v4.dart';
+import 'package:collection/collection.dart';
+import 'package:http/http.dart' as http;
+import 'package:path/path.dart' as p;
+
+// This examples walks through the creation of an S3 bucket and the process of
+// uploading a file to that bucket and retrieving a secure URL for reading back
```

Comment:
Nit: It's not wrong but probably best to use consistent language
```suggestion
// uploading a file to that bucket and retrieving a pre-signed URL for reading back
```